ruby/ext/google/protobuf_c/defs.c

```diff
@@ -250,7 +250,7 @@ void Descriptor_free(void* _self) {
                        &self->pb_serialize_handlers);
   }
   if (self->json_serialize_handlers) {
-    upb_handlers_unref(self->pb_serialize_handlers,
+    upb_handlers_unref(self->json_serialize_handlers,
```

Comment:
I would have expected a UPB_DEBUG_REFS build to catch this. I don't know if there's any easy way to do this, but if there was any way to test without NDEBUG defined, that would probably be a good idea.